docs/src/docs/asciidoc/getting-started.adoc

```diff
@@ -551,14 +578,17 @@ the resulting HTML files depends on whether you are using Maven or Gradle:
 
 |===
 
-The generated snippets can then be included in the manually created Asciidoctor file from
-above using the
+You can then include the generated snippets in the manually created Asciidoc file
+(described earlier in this section) by using the
 http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#include-files[include macro].
-The `snippets` attribute that is automatically set by `spring-restdocs-asciidoctor`
-configured in the <<getting-started-build-configuration, build
-configuration>> can be used to reference the snippets output directory. For example:
+You can use The `snippets` attribute that is automatically set by
```

Comment:
the

Reply:
Sigh. On the other hand, "The Snippets" makes a great band name. ;)